pages/contribute/editorial_board_guide.md

```diff
@@ -4,13 +4,101 @@ summary: This guide is there to help editors.
+For more information about this topic please visit the [Github documentation page](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
+
+## Create new page
+
+### Simple way: using the Github interface
+Simply copying the TEMPLATE file in the subdirectory and renaming it, is sufficient to generate a new page. To copy a template you have to:
+
+1. Go to the `TEMPLATE_` file of choice in the [github repo](https://github.com/elixir-europe/rdm-toolkit/tree/master/pages), every section has its own TEMPLATE file. For example the [TEMPLATE_your_problem.md](https://github.com/elixir-europe/rdm-toolkit/blob/master/pages/your_problem/TEMPLATE_your_problem.md) file
+
+2. Click "Raw" on the github page to open the file 'as is'
+    {% include image.html file="raw_github.png" inline=true alt="Raw button Github." %}
+
+3. Select and copy all the content
+
+4. Go back to the main section were you want to make the new page, in our example this will be in */pages/your_problem*. Click on `Add file` on the right followed up by `Create new file`.
+    {% include image.html file="create_new_file_github.png" inline=true alt="Create new file github." %}
+
+5. Paste the copied content from the template
+
+6. Name the file by choosing a unique self explaining short name. Without capitals and without spaces (replace them with underscores).
+    {% include image.html file="name_file_github.png" inline=true alt="Name the file in github." %}
+
+7. Check the frontmatter/metadata of the markdown page:
+    - delete `search: exclude` attribute
+    - add the names of the authors to the contributors list
+    - optional: changing the title to an appropriate one
+
+8. Commit to the master branch by clicking `Commit new file`
+     {% include image.html file="commit_to_master_github.png" inline=true alt="Commit new file in github." %}
+
+9. If the markdown file is named *example.md*, the page will be rendered at https://rdm.elixir-europe.org/example. This link can be provided to the contributor through the issue.
+
+### Advanced: Working on your own feature branch and pushing local changes
 
 Also as an editor, try to work as much as possible on a different branch than the master branch. This can be on the elixir-europe rdm-toolkit repo or your own fork. Open a pull request if you want to merge your changes with the master branch. This way it is possible for other editors to give feedback on your changes. Typos or other small fixes can off course be done immediately on the master branch. Don't forget to keep your branch/fork up to date with the changes in the master branch.
 
-This is a general workflow if you work on your own fork (copy) of the rdm-toolkit repo repo:
+This is a general workflow if you work on your own fork (copy) of the rdm-toolkit repo:
```

Comment:
Since this process of forking etc... is not specific for this repo but a generic way of working with GitHub, I would propose to wrap this text in a subpage and paste the link here. What do you think?